libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Logger/SFLogger.h

```diff
@@ -103,64 +118,64 @@
  */
 - (void)e:(nonnull Class)cls format:(nonnull NSString *)format, ...;
 
+
 /**
- * Logs a warning log line.
+ * Logs an info log line.
  *
  * @param cls Class.
  * @param message Log message.
  */
-- (void)w:(nonnull Class)cls message:(nonnull NSString *)message;
+- (void)i:(nonnull Class)cls message:(nonnull NSString *)message;
 
 /**
- * Logs a warning log line.
+ * Logs an info log line.
  *
  * @param cls Class.
  * @param format The format message, and optional arguments to expand in the format.
  * @param ... The arguments to the message format string.
  */
-- (void)w:(nonnull Class)cls format:(nonnull NSString *)format, ...;
+- (void)i:(nonnull Class)cls format:(nonnull NSString *)format, ...;
 
 /**
- * Logs an info log line.
+ * Logs a debug log line.
  *
  * @param cls Class.
  * @param message Log message.
  */
-- (void)i:(nonnull Class)cls message:(nonnull NSString *)message;
+- (void)d:(nonnull Class)cls message:(nonnull NSString *)message;
 
 /**
- * Logs an info log line.
+ * Logs a debug log line.
  *
  * @param cls Class.
  * @param format The format message, and optional arguments to expand in the format.
  * @param ... The arguments to the message format string.
  */
-- (void)i:(nonnull Class)cls format:(nonnull NSString *)format, ...;
+- (void)d:(nonnull Class)cls format:(nonnull NSString *)format, ...;
 
 /**
- * Logs a verbose log line.
+ * Logs a default log line.
```

Comment:
Should be warning instead of default

Reply:
@wmathurin There isn't an equivalent of warn. Warn is mapped to default level. From what I read, the default level  seems to be appropriate for warnings.